keyboards/kudox_game/keymaps/default/keymap.c

```diff
@@ -35,10 +44,31 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   //┌────────┼────────┼────────┼────────┼────────┼────────┼────────┐
      _______ ,_______ ,KC_LEFT ,KC_DOWN ,KC_RIGHT,KC_LBRC ,KC_RBRC ,
   //├────────┼────────┼────────┼────────┼────────┼────────┼────────┐
-     _______ ,KC_BSPC                   ,KC_CIRC ,KC_LPRN ,KC_RPRN ,
+     _______ ,KC_BSPC                   ,KC_DEL  ,KC_LPRN ,KC_RPRN ,
   //├────────┼────────┘                 └────────┴────────┼────────┼────────┐
      _______                                              ,_______ ,_______
   //└────────┘                                            └────────┴────────┘
+  ),
+
+  [_LIGHT] = LAYOUT(
+  //┌────────┬────────┬────────┬────────┬────────┬────────┐
+     _______ ,RGB_HUI ,RGB_HUD ,RGB_SAI ,RGB_SAD ,RGB_VAI ,
+  //└────────┼────────┼────────┼────────┼────────┼────────┤
+              RGB_M_P ,RGB_M_SW,RGB_M_X ,RGB_M_B ,RGB_VAD ,
+  //┌────────┼────────┼────────┼────────┼────────┼────────┼────────┐
+     _______ ,_______ ,RGB_M_SN,RGB_M_G ,RGB_M_R ,RGB_M_K ,RGB_M_T ,
+  //├────────┼────────┼────────┼────────┼────────┼────────┼────────┐
+     _______ ,RGB_M_T                   ,XXXXXXX ,RGB_VAI ,RGB_VAD ,
+  //├────────┼────────┘                 └────────┴────────┼────────┼────────┐
+     _______                                              ,RGB_MOD ,RGB_TOG
+  //└────────┘                                            └────────┴────────┘
   )
 
 };
+
+// Runs just one time when the keyboard initializes.
+#ifdef RGBLIGHT_ENABLE
+void matrix_init_user(void) {
```

Comment:
This needs to be the post-init function. If you use `matrix_init_user()` here, `rgblight_init()` will run afterwards, which will read the values set in EEPROM and override what you're doing here.

```suggestion
void keyboard_post_init_user(void) {
```

Reply:
Specifically, rgblight_init is called after `matrix_init`, which will wipe out any changes that you've made here.  However, `keyboard_post_init_*` is called after everything else, right before the main loop starts.  So this will get the functionality that you want.

Reply:
@noroadsleft @drashna 
Thank you for explaining it so courteously!
I understand the execution order of the initialization function and modified using `keyboard_post_init_user` from `matrix_init_user` .